tests/template_tests/filter_tests/test_yesno.py

```diff
@@ -1,6 +1,29 @@
 from django.template.defaultfilters import yesno
 from django.test import SimpleTestCase
 
+from ..utils import setup
+
+
+class YesNoTests(SimpleTestCase):
```

Comment:
There's already a single test for the `yesno`-filter in `django/tests/template_tests/syntax_tests/test_filter_syntax.py`. However, it's probably better to move them into a separate test-class. Please do this in a separate commit. Please see 7fce4dc5ff2d23bf43ee645f5bc0d6024ff1627f for an example.